blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: DialogSettings in Blazor RichTextEditor Component | Syncfusion
```

Comment:
Customization of Blazor Rich Text Editor Dialog  | Syncfusion

Reply:
addressed